hack/deployer/runner/gke_ssd_provisioner.go

```diff
@@ -0,0 +1,105 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package runner
+
+const GkeSsdProvisioner = `
```

Comment:
I'm not sure there are some benefits to save this content in an external file, it is not supposed to be customized and it is easier to embed it in the code.